pkl-core/src/test/files/LanguageSnippetTests/input/modules/invalidOutput1.pkl

```diff
@@ -0,0 +1 @@
+output: String = "abc"
```

Comment:
ideally this problem would be caught right here (override with different type), but I guess that's impossible for now

Reply:
We'd like to make Pkl stricter here and enforce that subclasses are also subtypes. Maybe something to do before we cut a 1.0 release.